feed/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Hi Bjørnar,  your code looks great so far, though I see the head is missing a meta description of the page, for SEO and page-readers. I think this is missing on the rest of your HTML as well. I am not sure if it matters as much in this CA, since it is about JS and not HTML.